ichnaea/scripts/importer.py

```diff
@@ -44,8 +43,9 @@ def load_file(session, source_file, batch_size=1000, userid=None):
                     # convert from unixtime to utc
                     time = datetime.datetime.utcfromtimestamp(time)
 
-                key = normalized_wifi_key(str(fields[1]))
-                if not valid_wifi_pattern(key):  # pragma: no cover
+                try:
+                    key = WifiKeyNode(String()).deserialize(str(fields[1]))
```

Comment:
I think we probably want to define an actual complete schema for the import data csv format here. If we have a full schema, this code might get less ugly - but that's a future todo.